### ISSUE TITLE
Issue/Feature Request: Typeahead popup style directive

### ISSUE THREAD

**Issue by d-nation** [angular-ui/bootstrap#3910]
Can you add a directive to change the left and top style attribute of the typeahead popup? Either that, or do what bootstrap does with the "popover" components where you can say that you want it left, middle, right, and up, down from the originating element.

Use case: The typeahead input is on the right side or bottom of the screen. Currently When the user starts typing, the list can be mostly off the screen to the right or bottom. It'd be nice to say "typeahead-popup-horizontal-position: 'middle'" or something to that nature.

**Comment by wesleycho** [comment]
Note: This is blocked by #3900 - once that feature makes it in in some capacity, then this will be possible.

**Comment by d-nation** [comment]
Fantastic! Thanks!

**Comment by icfantv** [comment]
Closing in favor of #4762.
